apps/penxle.com/src/lib/tiptap/components/TiptapRenderer.svelte

```diff
@@ -38,8 +36,10 @@
   });
 </script>
 
-<div bind:this={element} class={clsx(...(editor ? [] : ['ProseMirror', $$props.class]))} {...$$restProps}>
-  {#if !editor}
-    {@html html}
+<div bind:this={element} class={$$props.class} {...$$restProps}>
```

Comment:
```suggestion
<div bind:this={element} {...$$restProps}>
```

P2: 이거 확인해보았는데 restProps 만 넣어도 class 주입이 잘 됩니다!